content/300-guides/200-deployment/001-deploy-prisma.mdx

```diff
@@ -15,20 +15,26 @@ Projects using Prisma Client can be deployed to many different cloud platforms.
 
 Each paradigm has different tradeoffs that affect the performance, scalability, and operational costs of your application.
 
-Moreover, the user traffic pattern of your application is also an important factor to consider. For example, any application with consistent user traffic may be better suited for a [continuously running paradigm](#long-running-process-paas), whereas an application with sudden spikes may be better suited to [serverless](#serverless-faas).
+Moreover, the user traffic pattern of your application is also an important factor to consider. For example, any application with consistent user traffic may be better suited for a [continuously running paradigm](#long-running-process-paas), whereas an application with sudden spikes may be better suited to [serverless](#serverless-functions-faas).
 
 ### Long-running process (PaaS)
```

Comment:
I'm critical of using PaaS (and FaaS) as acronyms without introducing them anywhere. (PaaS in this instance does get expanded in the text, but FaaS doesn't in the next section.)